packages/core/src/index.ts

```diff
@@ -214,6 +214,8 @@ class Hopr<Chain extends HoprCoreConnector> extends EventEmitter {
         }
       },
       dialer: {
+        // Temporary fix
```

Comment:
For temporary fixes, can you link to the ticket that will fix them (and create if needed)

Reply:
Not implemented fits better at this point